updates/1/updates.sql

```diff
@@ -57,4 +57,52 @@ INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xA15C7Ebe1f07C
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xdfbc9050F5B01DF53512DCC39B4f2B2BBaCD517A", 8, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xdfbc9050F5B01DF53512DCC39B4f2B2BBaCD517A", "C", "Jobchain", "JOB", 1579390502, NULL, "jobchain", "JOB", "0xdfbc9050F5B01DF53512DCC39B4f2B2BBaCD517A");
+*
+UPDATE assets SET swapped_for="_ceth_0xdfbc9050F5B01DF53512DCC39B4f2B2BBaCD517A" WHERE identifier="_ceth_0x17280DA053596E097604839C61A2eF5efb7d493f";/*Introduce Jobchain token swap*/
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x17280DA053596E097604839C61A2eF5efb7d493f", 8, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x17280DA053596E097604839C61A2eF5efb7d493f", "C", "Jobchain", "JOB", 1552845272, '_ceth_0xdfbc9050F5B01DF53512DCC39B4f2B2BBaCD517A', "jobchain", "JOB", "0x17280DA053596E097604839C61A2eF5efb7d493f");
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xF9e5aF7B42D31D51677c75bbBD37c1986eC79AEE", 8, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xF9e5aF7B42D31D51677c75bbBD37c1986eC79AEE", "C", "QuickX Protocol", "QCX", 1548185499, NULL, "quickx-protocol", "QCX", "0xF9e5aF7B42D31D51677c75bbBD37c1986eC79AEE");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xAd4f86a25bbc20FfB751f2FAC312A0B4d8F88c64", 18, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xAd4f86a25bbc20FfB751f2FAC312A0B4d8F88c64", "C", "OptionRoom Token", "ROOM", 1611783103, NULL, "option-room", "", "0xAd4f86a25bbc20FfB751f2FAC312A0B4d8F88c64");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x0AeE8703D34DD9aE107386d3eFF22AE75Dd616D1", 18, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x0AeE8703D34DD9aE107386d3eFF22AE75Dd616D1", "C", "Tranche Finance", "SLICE", 1609247971, NULL, "tranche-finance", "SLICE", "0x0AeE8703D34DD9aE107386d3eFF22AE75Dd616D1");
+*
+UPDATE assets SET swapped_for="_ceth_0x0AeE8703D34DD9aE107386d3eFF22AE75Dd616D1" WHERE identifier="_ceth_0xa5Fd1A791C4dfcaacC963D4F73c6Ae5824149eA7";/*Introduce Tranche Finance token swap*/
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xa5Fd1A791C4dfcaacC963D4F73c6Ae5824149eA7", 18, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xa5Fd1A791C4dfcaacC963D4F73c6Ae5824149eA7", "C", "Jibrel Network Token", "JNT", 1552845272, '_ceth_0x0AeE8703D34DD9aE107386d3eFF22AE75Dd616D1', "jibrel", "JNT", "0xa5Fd1A791C4dfcaacC963D4F73c6Ae5824149eA7");
```

Comment:
Interestingly enough we have different data for the name and the started of the old token. Probably because of the cryptocompare start date. But your data is not wrong so I think we can leave it as it is: https://github.com/rotki/rotki/blob/79a4956d7a5c7579de53ca0a62882413a60331b7/rotkehlchen/data/all_assets.json#L6815-L6823

Reply:
I always check if there is already and entry. I might have searched for `Jibrel Network Token` and didn't find any result. For the next PRs I'll search for coingecko field

Reply:
Oh I would say search by address. It's the only thing guaranteed to be unique.